CHANGES.md

```diff
@@ -98,12 +98,14 @@ To be released.
  -  Since we decided to depend on TURN ([RFC 5766]) and STUN ([RFC 5389]) to
     work around NAT so that `Peer`'s endpoints don't have to be multiple,
     `Peer.Urls` was renamed to `Peer.EndPoint` and its type also was changed
-    from `IImmutableList<Uri>` to `IPEndPoint`.
-    [[#120], [#123] by Yang Chun Ung, [#126], [#127]]
+    from `IImmutableList<Uri>` to `DnsEndPoint`.
+    [[#120], [#123] by Yang Chun Ung, [#126], [#127], [#165]]
  -  `Swarm` became to ignore tip blocks of the same height (`Index`) that it
     already has and deal with only longer (higher) blocks.
  -  Fixed a bug that occured when `Swarm` was handling multiple responses at the
     same time.
+ -  Fixed a bug that hangs when calling the `Swarm` constructor in certain
+    runtimes. (e.g. Unity3d)
```

Comment:
```suggestion
    like Unity engine.
```

Unity seems to recently have changed their name from <q>Unity3D</q> to simply <q>Unity</q>.